src/EnergyPlus/DataSurfaces.hh

```diff
@@ -108,9 +108,9 @@ namespace DataSurfaces {
 
     enum class SurfaceClass : int
     {
-        Invalid = -1,
-        None,
-        Wall,
+        Invalid = -1, // If any addition classes get added to this list, add appropriate data
+        None,         // to ComputeNominalUwithConvCoeffs in DataHeatBalance.cc and make sure
+        Wall,         // that the data aligns with any revision to this enum.
```

Comment:
The comments here are a _little_ oddly placed.  I would have just:
```
enum class Foo {
  // comment 1
  // comment 2
  Invalid = -1,
  Bar,
  Num
}
```

Nothing to worry about for now, just noting.